lib/Doctrine/DBAL/Query/Expression/ExpressionBuilder.php

```diff
@@ -39,41 +39,61 @@ public function __construct(Connection $connection)
     }
 
     /**
-     * Creates a conjunction of the given boolean expressions.
+     * Creates a conjunction of the given expressions.
      *
      * Example:
      *
      *     [php]
      *     // (u.type = ?) AND (u.role = ?)
      *     $expr->andX('u.type = ?', 'u.role = ?'));
```

Comment:
Should we just get rid of this example? We're too used to getting being covered by static analysis to maintain code examples in comments.

Reply:
Good point, totally missed that. Fixed in 2c69e551c6138cf46fa7c6c3377ed1774e564f65.